packages/react-instantsearch-core/src/connectors/__tests__/connectNumericMenu.js

```diff
@@ -279,6 +279,34 @@ describe('connectNumericMenu', () => {
         '>=': [100],
         '<=': [200],
       });
+
+      params = connect.getSearchParameters(
```

Comment:
I feel like this should be its own test, what do you  think?

Reply:
I felt the same way, I wasn't sure how to name it without breaking up the whole test thing into tests like:
```js
 it('refines the numeric facet for 100:200', () = > {
      params = connect.getSearchParameters(
        initSP,
        { attribute: 'facet', contextValue },
        { multiRange: { facet: '100:200' } }
      );
      expect(params.getNumericRefinements('facet')).toEqual({
        '>=': [100],
        '<=': [200],
      });
  }
```

Reply:
`refines the corresponding numeric facet` and `refines the corresponding numeric facet with value 0`